src/types/common.d.ts

```diff
@@ -0,0 +1,22 @@
+export interface MusicAccidental {
+  note: number;
+  accidental: number;
+}
+
+export interface MusicNoteParts {
+  root: string;
+  accidental: string;
+  type?: string;
+}
+
+/** Range 0 - 11 */
+export type KeyValue = number;
+/** Range 0 - 6 */
```

Comment:
This comment will not force typescript for a compile error will it?

Reply:
No, will only provide info in the editor
![image](https://user-images.githubusercontent.com/22865285/111124692-141a5200-8571-11eb-861c-e8c494060984.png)

Reply:
with the mouse over it

Reply:
Agree no need for that